.github/workflows/lint.yml

```diff
@@ -0,0 +1,15 @@
+name: lint
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
```

Comment:
so, that didn't work, since we're developing using pyenv, and pyenv uses the `.python-version` file as well, meaning that it would conflict if we set it to 3.9 and then during development we want to try 3.10 and so on. I'll leave it hardcoded in all the tasks for now. It's easy to change later if we need to change the version, just a search replace.

Reply:
I just realized that I can use `python-version-file` to choose the filename, so I'll just call it something else and then we're all good :smile: